src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/DefaultTranscriptsList.scss

```diff
@@ -82,3 +82,13 @@
 .hidden {
   display: none;
 }
+
+.filterSortIndicator {
```

Comment:
I wonder if there may be a better name for this. `activeIndicator`, `activityIndicator`, `onIndicator`?

Reply:
I replaced it with `activeIndicator`

Reply:
✅